_setup.sh

```diff
@@ -1,9 +1,9 @@
+#!/usr/bin/env sh
+
 function build() {
   npm install
   composer install
 
-  vendor/bin/phpcs --config-set installed_paths ../../wp-coding-standards/wpcs 
-
   wp core download
```

Comment:
This step failed when I tried to initialize it on a certain site. We need to handle if wp-cli is installed case.